### PR TITLE
v1.1.0: Roll up

### PR DESCRIPTION
**Pull request by tdewey-rpi**

### CODE REVIEW

File: key-writer/keywriter.sh [hunk 1]
```diff
@@ -159,15 +161,15 @@ BOOTCODE_BINARY_IMAGE=
 BOOTCODE_FLASHING_NAME=
 case ${RPI_DEVICE_FAMILY} in
     4)
-        SOURCE_EEPROM_IMAGE="/lib/firmware/raspberrypi/bootloader-2711/latest/pieeprom-2024-05-17.bin"
-        BOOTCODE_BINARY_IMAGE="/var/lib/rpi-sb-provisioner/recovery.bin"
+        SOURCE_EEPROM_IMAGE="/lib/firmware/raspberrypi/bootloader-2711/latest/pieeprom-2024-07-30.bin"
```
Comment (timg236) [inline key-writer/keywriter.sh:164]: I've just released pieeprom-2024-09-10.bin for 2712 - that's also a default/automatic release

Comment (tdewey-rpi) [reply key-writer/keywriter.sh:164]: > I've just released pieeprom-2024-09-10.bin for 2712 - that's also a default/automatic release

I only see a 2024-10-09.bin for 2712 - I'm guessing a new renamed release is coming.

Comment (timg236) [reply key-writer/keywriter.sh:164]: ACK .... unless I just leave it thirty days and fixup the commit message ;-)

File: debian/postinst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ ! $(getent group rpi-sb-provisioner) ]; then
+if [ ! "$(getent group rpi-sb-provisioner)" ]; then
```
[the remark this answers is not in the document]
Comment (tdewey-rpi) [reply debian/postinst:3]: Agreed - now we're in the devscripts universe, we should consider expanding our use of the debhelpers.

**Pull request merged**